.travis.yml

```diff
@@ -106,6 +106,30 @@ matrix:
         - ~/bin/bazel test --test_output=all ${OLC_PATH}:all
         - cd ${OLC_PATH} && bash format_check.sh
 
+    # Python implementation. Lives in python/, tested with bazel.
+    - language: python
+      python: 3.7
+      dist: bionic
```

Comment:
What's prompting this change? We currently have the dist set to trusty globally - do Python 3.7 and 3.8 need to specifically run with bionic?

Reply:
Yes, trusty does not support python 3.7 or 3.8. The Travis build for the first commit failed on [Python 3.7](https://travis-ci.org/google/open-location-code/jobs/630116467) and on [Python 3.8](https://travis-ci.org/google/open-location-code/jobs/630116468) with the reason: `Unable to download 3.(7|8) archive. The archive may not exist. Please consider a different version.`

Reply:
Got it, thanks for the clarification. Would you mind setting the global dist to bionic and seeing if everything still passes? I'd prefer to keep everything in sync if possible.

Reply:
Will do. 👍

Reply:
It appears that Java 8 is no longer available on [bionic](https://travis-ci.org/google/open-location-code/jobs/630126866), as it is EOL. Can bump the dist down to xenial, or set it specifically for java8 and add later jdks to be sure they work as well.

Reply:
I like your second approach - let's default to bionic and set it to older dist versions as needed. Thanks for doing this!